packages/js/api/src/models/products/abstract/inventory.ts

```diff
@@ -56,6 +56,13 @@ abstract class AbstractProductInventory extends Model {
 	 * @type {boolean}
 	 */
 	public readonly isOnBackorder: boolean = false;
+
+	/**
+	 * Indicates when the threshold for when low stock notification will be sent to the store admin.
```

Comment:
Just a minor suggested wording change:

```suggestion
	 * Indicates the threshold for when the low stock notification will be sent to the store admin.
```

We may also want to consider swapping `store admin` to `merchant` to match what we use in other end-to-end tests and in the utils package.

Reply:
Makes sense to be consistent. I've made the change based on your suggestion.